client/src/unifyfs-sysio.c

```diff
@@ -998,7 +997,7 @@ ssize_t UNIFYFS_WRAP(read)(int fd, void* buf, size_t count)
 /* TODO: find right place to msync spillover mapping */
 ssize_t UNIFYFS_WRAP(write)(int fd, const void* buf, size_t count)
 {
-    LOGDBG("write was called for fd %d", fd);
+    LOGDBG("write %d bytes to fd %d", count, fd);
```

Comment:
should use `%zu` for size_t `count` parameter

Reply:
Ok, will fix